wren-launcher/utils/docker.go

```diff
@@ -306,8 +310,35 @@ func CheckWrenAIStarted(url string) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
+		fmt.Println("WrenUI is not ready yet")
 		return fmt.Errorf("WrenAI is not started yet")
 	}
+	return nil
+}
+
+func CheckAIServiceStarted(projectDir string) error {
+	// check response from localhost:5555
+	envExampleFile := getEnvFilePath(projectDir)
+
+	// read ai port from env file
+	envFileContent, err := os.ReadFile(envExampleFile)
+	if err != nil {
+		return err
+	}
+	reg := regexp.MustCompile(`WREN_AI_SERVICE_PORT=(.*)`)
+	aiPort := reg.FindStringSubmatch(string(envFileContent))[1]
 
+	// health check
+	url := fmt.Sprintf("http://localhost:%s/health", aiPort)
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		fmt.Println("AI service is not ready yet")
```

Comment:
maybe remove this as well

Reply:
delivered in 38544ea2cca6541252f3c7c738bd28b0d0526e93